lib/ao_host.js

```diff
@@ -79,7 +77,9 @@ class AOHost extends AsyncEventEmitter {
 
     this.adapter.once('order:snapshot', (snapshot) => {
       this.orderSnapshot = snapshot
-      this.loadAllAOs().then(() => {
+      this.loadAllAOs().then(() => { // eslint-disable-line
+        this.emit('ready')
```

Comment:
Could reject orders if not ready?

Reply:
Might not be that useful actually, Im assuming the registration only takes milliseconds